src/main/java/org/zalando/nakadi/config/NakadiConfig.java

```diff
@@ -60,4 +66,24 @@ public ApplicationService applicationService(@Value("${nakadi.auth.plugin.factor
         }
     }
 
+    @Bean
+    @Qualifier("default_storage")
+    public Storage defaultStorage(final StorageDbRepository storageDbRepository,
+                                  final Environment environment) throws InternalNakadiException {
+        final Storage storage = new Storage();
+        storage.setId("default");
+        storage.setType(Storage.Type.KAFKA);
+        storage.setConfiguration(new Storage.KafkaConfiguration(
+                environment.getProperty("nakadi.zookeeper.exhibitor.brokers"),
+                Integer.valueOf(environment.getProperty("nakadi.zookeeper.exhibitor.port", "0")),
+                environment.getProperty("nakadi.zookeeper.brokers"),
+                environment.getProperty("nakadi.zookeeper.kafkaNamespace", "")));
+        try {
+            storageDbRepository.createStorage(storage);
```

Comment:
Why do we need to have the default storage in the database if the application only uses it's in memory representation?